packages/dicomImageLoader/src/imageLoader/isColorConversionRequirementsFulfilled.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * This function checks color space conversion data requirements before
+ * apply them
+ * @param imageFrame
+ * @param RGBA
+ * @returns
+ */
+export default function isColorConversionRequirementsFulfilled(
```

Comment:
@wayfarer3130 Can you please look at this function? Thanks

Reply:
I will limit the checks to only YBR_FULL_422